examples/docker/README.rst

```diff
@@ -50,6 +50,13 @@ matches the name of the image referenced in the ``MLproject`` file.
 
 Finally, run the example project using ``mlflow run examples/docker -P alpha=0.5``.
 
+.. note::
+    If running this example on M1 / M2 Mac, ensure that Docker Desktop is running and that you are
+    logged in to the Docker service. It is recommended to run this example with the optional
+    'Rosetta' compatibility mode (configured in settings) to ensure that if you make modifications
```

Comment:
Can we link a doc on how to set up Rosseta?

Reply:
Is this the doc?

https://docs.docker.com/desktop/install/mac-install/#system-requirements

Reply:
https://docs.docker.com/desktop/settings/mac/#features-in-development